.github/ISSUE_TEMPLATE/suggestion.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Suggestion
+about: Suggest an improvement or a feature
```

Comment:
Could we rename this template "Feature Request"?

Reply:
@aciccarello actually I was hesitating for minutes on that issue. For me, "Feature Request" sounds like... a narrower name. For example, #898, #935, #917 could be suggestions, but not feature requests. Feature requests sound like only an improvement only in typedoc code to me. Is that the only case for me?

If you get what I mean, "Suggestion" sounds like a superset of "Feature Request", which might embrace more issues.

What are your thoughts?

Reply:
That's a good point. I still lean towards "Feature Request" since it is a more standard term and fits the contents of the template better. We could possibly add a more general option in the future.

Reply:
Yeap!